src/collapse/utils.ts

```diff
@@ -0,0 +1,9 @@
+export function getElementHeight(element: HTMLElement | null) {
```

Comment:
You can maybe use this helper instead https://github.com/JetBrains/ring-ui/blob/6d38612f9deb7aa99a7a2e5b135fad920df8fdb4/src/global/dom.ts#L29

```
   getRect(element).height
```

Reply:
@andrey-skl Ah, I didn't know about this util. Yes, it can replace my custom getElementHeight util. Thank you!